CMakeLists.txt

```diff
@@ -761,6 +761,23 @@ endif()
 
 libical_option(ICAL_BUILD_DOCS "Build documentation" True)
 if(ICAL_BUILD_DOCS)
+  libical_option(ICAL_GLIB_BUILD_DOCS "Build libical-glib documentation" True)
```

Comment:
At first I was against making the default True.   but these days, especially with vcpkg support (Windows), I think this is fine.

But I think then we also need to make GOBJECT_INTROSPECTION try by default

Reply:
Made it default to True